.gitignore

```diff
@@ -11,3 +11,4 @@ coverage
 *.log
 
 package-lock.json
+.vscode
```

Comment:
This changes deleted some .vscode files included by one of the authers. Are you sure this is something we want?

Reply:
I think this will only delete cached files in git, local files should not affect by this.

https://git-scm.com/docs/git-rm#Documentation/git-rm.txt---cached
```
--cached
Use this option to unstage and remove paths only from the index. Working tree files, whether modified or not, will be left alone.
```

Reply:
Check your changes, you deleted files.

Reply:
![image](https://user-images.githubusercontent.com/20819332/116157420-dc143c00-a6e4-11eb-827b-087998255b20.png)